R/predict_classprob.R

```diff
@@ -46,17 +50,16 @@ predict_classprob.model_fit <- function(object, new_data, ...) {
 # @keywords internal
 # @rdname other_predict
 # @inheritParams predict.model_fit
-predict_classprob <- function(object, ...)
+predict_classprob <- function(object, ...) {
   UseMethod("predict_classprob")
+}
 
 check_spec_levels <- function(spec) {
   if ("class" %in% spec$lvl) {
-    rlang::abort(
-      glue::glue(
-        "The outcome variable `{spec$preproc$y_var}` has a level called 'class'. ",
-        "This value is reserved for parsnip's classification internals; please ",
-        "change the levels, perhaps with `forcats::fct_relevel()`."
-      ),
+    cli::cli_abort(
+      "The outcome variable {.var {spec$preproc$y_var}} has a level called {.val class}.
+      This value is reserved for parsnip's classification internals; please
+      change the levels, perhaps with {.fn forcats::fct_relevel}.",
```

Comment:
```suggestion
      c(
        "The outcome variable {.var {spec$preproc$y_var}} has a level called {.val class}.",
        "i" = "This value is reserved for parsnip's classification internals; please 
               change the levels, perhaps with {.fn forcats::fct_relevel}."
      ),
```

Making use of cli's newline formatting happiness. TIL about `.fn` as an alias for `.fun`! Nice!